mscore/mixer.cpp

```diff
@@ -125,6 +126,24 @@ Mixer::Mixer(QWidget* parent)
       retranslate(true);
       }
 
+
+
+//---------------------------------------------------------
+//   showDetailsToggled
+//---------------------------------------------------------
+
+void Mixer::toggleMuteChannels(){
```

Comment:
Please read [MuseScore coding rules](https://musescore.org/en/handbook/developers-handbook/finding-your-way-around/musescore-coding-rules) and make the necessary adjustments. Primarily "Banner Style".

Reply:
I saw an adjustment to follow banner style, but most of the code does not matches the whole coding style. Do a careful read over and match all styles. 6 space indents, curly braces on the line except for functions and classes, etc. Otherwise, you'll get a whole list of comments nitpicking each line :-)

Reply:
reformatted all using the qtcreator style provided